### DISCUSSION TITLE
导入黑名单并移除播放器页相对应视频推荐

### DISCUSSION THREAD

**Discussion by xx146542xx** [the1812/Bilibili-Evolved#5054] [功能建议 / Ideas]
黑名单功能残废有段时间了 总不能老让那些我不喜欢的up在推荐视频里面拉屎撒尿吧

**Comment by Geeyun-JY3** [comment]
<https://github.com/the1812/Bilibili-Evolved/discussions/1301#discussioncomment-8787140>：

> # 不会做的功能
> 以下的功能就别提功能建议了, 不会做的.
> 
> - ……
> - 屏蔽 / 筛选某某 UP 主的视频
> - 屏蔽 / 筛选满足某些条件的视频
